networking/k8s_nmstate/k8s-nmstate-using-kubernetes-nmstate.adoc

```diff
@@ -0,0 +1,20 @@
+[id="k8s-nmstate-using-kubernetes-nmstate"]
+
+= Using Kubernetes NMState
+include::modules/common-attributes.adoc[]
+:context: k8s-nmstate-using
+
+The `kubernetes-nmstate` operator provides a Kubernetes API for peforming state-driven network configuration across the {product-title} cluster's nodes with `nmstate`. `kubernetes-nmstate` provides users with functionality to configure various network interface types, DNS and routing on cluster nodes. Additionally, the daemons on the cluster nodes periodically report on the state of each node's network interfaces to the API server.
+
+
+[NOTE]
+.Technology Preview
+====
+The `kubernetes-nmstate` operator is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+
+`nmstate` requires Network Manager installed on each cluster node.
```

Comment:
I see both Network Manager and NetworkManager usage, so we just need to be consistent.